streamer/configuration.py

```diff
@@ -238,23 +247,26 @@ def get_type_name_static(type: Optional[Type],
     if type is str:
       # Call it a string, not a "str".
       return 'string'
-    elif type is list:
+    if type is list:
```

Comment:
I know some styles (and even Google styles) say to not use "else if" or elif when the block above returns.  But I think it is more clear to have this as an elif chain so it is clear only one of all these will be hit.

Reply:
ok ill revert.